great_expectations/checkpoint/util.py

```diff
@@ -205,31 +203,66 @@ def send_email(
         return "success"
 
 
+def get_substituted_validation_dict(
+    substituted_runtime_config: dict, validation_dict: dict
+) -> dict:
+    substituted_validation_dict = {
+        "batch_request": get_substituted_batch_request(
+            substituted_runtime_config=substituted_runtime_config,
+            validation_batch_request=validation_dict.get("batch_request"),
+        ),
+        "expectation_suite_name": validation_dict.get("expectation_suite_name")
+        or substituted_runtime_config.get("expectation_suite_name"),
+        "expectation_suite_ge_cloud_id": validation_dict.get(
+            "expectation_suite_ge_cloud_id"
```

Comment:
Same here. Perhaps a ticket in the backlog?

Reply:
This was an older code, which made sense to me, so I did not change it.